### DISCUSSION TITLE
The "Tag" dialog should have a "Related" link like all other more-info dialogs

### DISCUSSION THREAD

**Discussion by seidler2547** [home-assistant/frontend#20994] [Other feature requests] [answered]
In all "more-info" popup cards there is a menu item "Related" which shows Integration, Device, Area and "Part of the following automation". This entry does not exist for the Tag dialog. This makes it almost impossible to find out in which automation a tag is used.

**Comment by karwosts** [comment, marked as answer]
I think this will be available in 2024.6:

https://rc.home-assistant.io/blog/2024/05/29/release-20246/#tag-entities

**Reply by seidler2547** [reply]
Crazy how this has bothered me since ... forever and the day I post about it it gets solved :smile:

**Reply by SeanPM5** [reply]
I don't think it's fully solved just yet. While the tag entity is a welcome addition, the related section is still going to be blank for everyone because the automation triggers currently reference the tag ID directly and _not_ the actual tag entity.

<img width="440" alt="Screen Shot 2024-06-05 at 3 23 36 PM" src="https://github.com/home-assistant/frontend/assets/9065557/b3ac2301-2ed3-4667-9194-ce6d2a9f26c2">

**Reply by karwosts** [reply]
For what it's worth the list of Related comes from core, so if this was to be improved for tag triggers, that would have to be a core change, not frontend.
